app/Domain/Integrations/Controllers/IntegrationController.php

```diff
@@ -311,6 +329,28 @@ public function storeKeyVisibilityUpgrade(string $id, KeyVisibilityUpgradeReques
         );
     }
 
+    public function getOrganizers(string $id, GetOrganizersRequest $request): RedirectResponse
```

Comment:
I think it's better to move this to a dedicated class `OrganizerController` and change the endpoint to `GET /organizers?name={name}`